StoreSpidLogs/index.ts

```diff
@@ -82,7 +92,10 @@ export async function index(
       },
       spidBlobItem => {
         return {
-          spidRequestResponse: spidBlobItem
+          spidRequestResponse: encryptWithRsaPublicKey(

```

Comment:
I've thinked about this, it's better to store a json with all the metadata (ip, createdat, fiscalcode) in plaintext properties and two encrypted fields (requestPayload, responsePayload)